crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -966,6 +898,129 @@ impl Target {
 
 // Rust native methods
 impl Target {
+    /// Adds a [PackedOperation] to the [Target].
+    ///
+    /// Said addition results in a [NormalOperation] in the [Target] as variadics
```

Comment:
Can you use a normal rustdoc formatting here for the arguments and return type? This is our internal public interface for adding instructions so having a built more detail and an example would be good.